api_specs/v2/component_analyses.yaml

```diff
@@ -419,14 +551,26 @@ components:
                   type: string
                 version:
                   type: string
+                used_packages:
```

Comment:
Can this be renamed to sub_packages?

Reply:
This can have "Modules" also. So the word "sub-package" is not correct in that context. I also could not get the right word for this attribute. Please suggest if something else can be used. The content of this attribute is the list of Modules/packages used in the source code, we need to derive something that indicates this usage.

Reply:
As discussed modified 'used-packages' to 'included-packages' and introduced 'root_package_used' flag for request to make it symmetric with response.